examples/example-clientlib-ktx-app/app/src/main/java/com/solanamobile/ktxclientsample/viewmodel/SampleViewModel.kt

```diff
@@ -98,37 +99,39 @@ class SampleViewModel @Inject constructor(
                 }
             }
 
-            persistanceUseCase.persistConnection(
-                currentConn.publicKey,
-                currentConn.accountLabel,
-                currentConn.authToken
-            )
+            result.successPayload?.let { currentConn ->
```

Comment:
This convenience property allows consumers to access the payload without having to create an if/then or when block handling all types of the sealed class.